README.md

```diff
@@ -169,6 +177,43 @@ Found 2 known vulnerabilities in 1 packages
 ]
 ```
 
+## Security Model
+
+This section exists to describe the security assumptions you **can** and **must not**
+make when using `pip-audit`.
+
+TL;DR: **If you wouldn't `pip install` it, you should not `pip audit` it.**
+
+`pip-audit` is a tool for auditing Python environments for packages with
+*known vulnerabilities*. A "known vulnerability" is a flaw in a package that,
+if uncorrected, *might* allow a malicious actor to perform unintended actions.
```

Comment:
...and also has been publicly reported to the project maintainers.

Reply:
1b0e38f